src/plone/api/env.py

```diff
@@ -201,6 +202,20 @@ def test_mode():
     return IS_TEST
 
 
+def read_only_mode():
+    """Returns True if you are running the zope instance on an read only ZODB.
```

Comment:
IMO, it would be better: "Check if the Zope instance is running on a read-only ZODB."